libs/sdk-ui-theme-provider/src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -27,6 +27,17 @@ export interface IThemeProviderProps {
      * component tree.
      */
     workspace?: string;
+
+    /**
+     * Theme object to use instead of one loaded from workspace
+     */
+    adhocTheme?: ITheme;
+
+    /**
+     * If true, global theme will be automatically cleared during component unmount.
+     * Default value is false
+     */
+    autoClear?: boolean;
```

Comment:
do we need this? if the correct behavior is to always clear stuff after itself, then why allowing not to do it? 

it seems that it leaks impl detail - that there is something stateful about the theme.

would this autoClear make sense if we had just styled components all way through?

I suggest to remove this and always clear after itself..

Reply:
removed and done by default